installer/uninstaller.sh

```diff
@@ -43,25 +43,45 @@ helm_arr=()
 K8S_NS_ARR=()
 
 cd ../scripts
-./imm -k rm t default-tenant
-cd ../installer
+if [[ "$quiet" != "yes" ]]; then
+    echo "Uninstalling IMM. Following components will be removed:"
+    echo "*Default tenant with it's resources(models,endpoints etc.)"
+    echo "*Helm releases (mgt-api, crd etc.)"
```

Comment:
Wouldn't be better to just print here all these releases?

Reply:
It's just for preview. Releases to be deleted are printed later during uninstallation and user must once more confirm that it's desired action.

Reply:
But user in this moment is not aware that all resources will be printed once again before the deletion will start. For better UX I suggest to print it here